devel/hwloc/Portfile

```diff
@@ -22,10 +22,30 @@ long_description \
 homepage            http://www.open-mpi.org/projects/hwloc/
 master_sites        http://www.open-mpi.org/software/hwloc/v${branch}/downloads/
 
-checksums           rmd160  c6a991bea0c7b4271bb1452ee261a58688179367 \
-                    sha256  8af89b1164a330e36d18210360ea9bb305e19f9773d1c882855d261a13054ea8
+checksums           rmd160  add29745a69a7ef7f9c35bb067e1858174e07f55 \
+                    sha256  1fb8cc1438de548e16ec3bb9e4b2abb9f7ce5656f71c0906583819fcfa8c2031 \
+                    size    6761831
 
-configure.args      --without-x
+depends_build       port:pkgconfig
+depends_lib-append  port:libxml2
+configure.args      --without-x --disable-cairo
 
 livecheck.type      regex
 livecheck.regex     "${name} v(\[0-9.\]+) "
+
+variant gui description {Add graphical output capability} {
+    depends_lib-append      path:lib/pkgconfig/cairo.pc:cairo \
+                            port:xorg-libX11 \
+                            port:xorg-libice \
+                            port:xorg-libsm \
+                            port:openssl
+    require_active_variants path:lib/pkgconfig/cairo.pc:cairo x11
+    configure.args-delete   --without-x --disable-cairo
```

Comment:
I assume that there are the opposite commands that need to be added? i.e `--with-x` and `--enable-cairo`.